analytics/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+import dependencies.Dependencies
+import dependencies.Versions
+
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
+}
+
+apply(from = "../config/quality.gradle")
+
+android {
+    compileSdk = Versions.android_compile_sdk
+
+    defaultConfig {
+        minSdk = Versions.android_min_sdk
+        targetSdk = Versions.android_target_sdk
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        getByName("release") {
```

Comment:
Can this not be `release {`? Same goes for the other modules.

Reply:
Right, we can use that. I don't remember why I thought it was not possible. Fixed.